RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2512,9 +2509,9 @@ boolean L11_shenStartQuest()
 	{
 		return false;
 	}
-	if (my_daycount() < 2 || !allowSoftblockShen())
+	
+	if (!allowSoftblockShen())
```

Comment:
I don't think we need this any more either. Otherwise it's going to do everything other than speak to Shen and quests that are in days 1-3 snake zones, then speak to Shen after the softblock is released.
Speaking to him as soon as we can will allow us to potentially unblock many more quests.
I would remove all traces of `allowSoftblockShen()` and `setSoftblockShen()` (they're defined in auto_util.ash).